catalog/app/components/PaymentDialog/messages.js

```diff
@@ -4,24 +4,28 @@ import { defineMessages } from 'react-intl';
 export default defineMessages({
   already: {
     id: 'app.containers.Profile.alreadyFree',
-    defaultMessage: 'Already on plan',
+    defaultMessage: 'Nothing to do',
   },
   free: {
     id: 'app.containers.Profile.free',
     defaultMessage: 'Free',
   },
-  business_monthly_490: {
```

Comment:
I guess it doesn't matter much given that no one is using the business plan... But in general, we shouldn't assume that backend and frontend changes will go out simultaneously.

Reply:
sure though in this case there is no way that `business...` could ever be selected by the frontend (or the backend as we never allowed people to elect it). i guess a backend could return it

Reply:
Actually, you _can_ upgrade to business right now.

Reply:
you are right. damn. upgrade with you 💳 :p